src/main/java/core/basesyntax/service/impl/WriterToCsvImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.Writer;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class WriterToCsvImpl implements Writer {
```

Comment:
```suggestion
public class WriterImpl implements Writer {
```
your writer doesnt have any specific logic for csv format